packages/orchestrator-ui-components/src/components/WfoForms/formFields/SubscriptionField.tsx

```diff
@@ -0,0 +1,380 @@
+/*
+ * Copyright 2019-2023 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import React from 'react';
+import { css } from '@emotion/react';
+import {
+    EuiButtonIcon,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiFormRow,
+    EuiText,
+} from '@elastic/eui';
+import get from 'lodash/get';
+import ReactSelect from 'react-select';
+import { useTranslations } from 'next-intl';
+import {
+    connectField,
+    filterDOMProps,
+    joinName,
+    useField,
+    useForm,
+} from 'uniforms';
+
+import { useOrchestratorTheme } from '../../../hooks';
+import { FieldProps, Option, PortMode, ProductTag } from './types';
+import { getPortMode } from './utils';
+import { useGetSubscriptionDropdownOptions } from '../../../hooks/surf/useGetSubscriptionDropdownOptions';
+import { SubscriptionDropdownOption } from '../../../types';
+import { getReactSelectInnerComponentStyles } from './reactSelectStyles';
+
+const subscriptionFieldStyling = css`
```

Comment:
Try to have the styling out of the component

Reply:
Moved it back to its own file